admin_guide/manage_users.adoc

```diff
@@ -14,19 +14,57 @@ toc::[]
 [[managing-users-overview]]
 == Overview
 
+A user is an entity that interacts with the {product-title} API. These can be a developer for developing applications or a administrator for managing the cluster. Users can be assigned to groups, which set the permissions applied to all the group's members. For example, you can give API access to a group, which give all members of the group API access.
+
 This topic describes the management of
-xref:../architecture/core_concepts/projects_and_users.adoc#architecture-core-concepts-projects-and-users[user] accounts,
-including how new user accounts are created in {product-title} and how they can
-be deleted.
+xref:../architecture/core_concepts/projects_and_users.adoc#architecture-core-concepts-projects-and-users[user]
+accounts, including how new user accounts are created in {product-title} and how
+they can be deleted.
 
 [[managing-users-adding-a-user]]
-== Adding a User
+== Creating a User
+
+The process for creating a user depends on the configured
+xref:../install_config/configuring_authentication.adoc#identity-providers[identity
+provider]. By default, {product-title} uses the `DenyAll` identity provider,
+which denies access for all user names and passwords. 
 
-After new users log in to {product-title}, an account is created for that user
-per the xref:../install_config/configuring_authentication.adoc#identity-providers[identity
-provider] configured on the master. The cluster administrator can
-xref:../admin_guide/manage_rbac.adoc#admin-guide-manage-rbac[manage the access level of
-each user].
+The following process creates a new user, then adds a role to the user:
+
+. Create the user account depending on your identity provider. This can depend
+on the `mappingmethod` used as part of the identity provider configuration. See
+the
+xref:../install_config/configuring_authentication.adoc#mapping-identities-to-users[Mapping
+Identities to Users] section for more information.
+
+. Give the new user the desired role:
++
+----
+# oc adm policy add-cluster-role-to-user <role> <user>
```

Comment:
I think we should prefer `oc create rolebinding`.  @openshift/sig-security probably has an opinion, but for creation I'm hard pressed to think of a reason for ambiguity in the created rolebinding.

Reply:
Yes `oc create clusterrolebinding` is preferred, but if `oc adm policy add-cluster-role-to-user` is "better" for some reason then you should definitely suggest the use of the --rolebinding-name option as well.

Reply:
I agree with @deads2k and @simo5 - we need to start pushing people towards the explicit commands.  Otherwise they end up with the wrong conceptual model for how RBAC works.